SnapMD.ConnectedCare.ApiModels/EmailInvitationRequest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SnapMD.ConnectedCare.ApiModels
+{
+    public class EmailInvitationRequest
+    {
+        public int? HospitalId { get; set; }
+
+        public int? UserId { get; set; }
```

Comment:
There is an older api sending invitation to the couser.  I use this for
both api request.
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
This
email has been sent from a virus-free computer protected by Avast.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DDB4FAA8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Tue, Jan 26, 2016 at 11:32 AM, Aaron <notifications@github.com> wrote:

> In SnapMD.ConnectedCare.ApiModels/EmailInvitationRequest.cs
> <https://github.com/SnapMD/connectedcare-sdk/pull/54#discussion_r50886626>
> :
>
> > @@ -0,0 +1,21 @@
> > +﻿using System;
> > +using System.Collections.Generic;
> > +using System.Linq;
> > +using System.Text;
> > +using System.Threading.Tasks;
> > +
> > +namespace SnapMD.ConnectedCare.ApiModels
> > +{
> > +    public class EmailInvitationRequest
> > +    {
> > +        public int? HospitalId { get; set; }
> > +
> > +        public int? UserId { get; set; }
>
> @jay-lee00 <https://github.com/jay-lee00> Why do we need hospital ID and
> user ID?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SnapMD/connectedcare-sdk/pull/54/files#r50886626>.
>